django_project/core/settings/project.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 from .contrib import *  # noqa
 
+# Exclude elastic result which have a _score less than the minimum specified in min_score
+ELASTIC_MIN_SCORE = 3.5
```

Comment:
Basically we change how we query the campus, previously we used this query string : 
`"query": "(campus_provider: TVET OR campus_campus: TVET) AND (courses_isnull: false)",` And strangely the results is not good even I lowered the score to 1, it's like the OR is not working properly. So I found that the _long_description field in Campus model is a combination of campus and provider name, so I decided to query against that field instead. And suddenly I got a lot of results, for example 'TVET' with previous method I only got 3 result and with the new method I got more than 300, and most of them correct.
That's why I decided increased to 3.5, because now I got better results.